raiden/transfer/token_network.py

```diff
@@ -41,18 +44,27 @@ def subdispatch_to_channel_by_id_and_address(
 
     ids_to_channels = token_network_state.channelidentifiers_to_channels
 
-    channel_state = None
+    channel_states = []
     if node_address is not None:
         if node_address in ids_to_channels:
             channel_state = ids_to_channels[node_address].get(state_change.channel_identifier)
+            channel_states.append(channel_state)
+            if channel_state.both_participants_are_light_clients\
+                and type(state_change) in DuplicableStateChanges:
+                partner_channel_state = token_network_state.channelidentifiers_to_channels[channel_state.partner_state.address].get(
+                    channel_state.identifier
+                )
```

Comment:
this indentation is too confusing and the line too long, may i suggest:

```suggestion
                partner_channel = token_network_state.channelidentifiers_to_channels[channel_state.partner_state.address]
                partner_channel_state = partner_channel.get(channel_state.identifier)
```